orangecontrib/text/import_documents.py

```diff
@@ -261,19 +369,31 @@ def _read_text_data(self):
                 else UrlReader(path)
             text, error = reader.read()
             if text is not None:
-                text_data.append(text)
+                if type(reader) == ConlluReader:
+                    self.is_conllu = True
+                    for t in text:
+                        text_data.append(t)
+                    tokens.extend(reader.tokens)
+                    pos.extend(reader.pos)
+                    ner.extend(reader.ner)
+                else:
+                    self.is_conllu = False
+                    text_data.append(text)
                 batch.append(text_data)
             else:
                 errors.append(error)
 
             if self.cancelled:
                 return
 
+        self.tokens = tokens
+        self.pos = pos
+        self.ner = ner
```

Comment:
I hope that I didn't miss anything here. I would not put tokens, pos and ner to `self` and just return them. Since `run` is the only function calling `_read_text_data` at the end returns them. And I don't see that they would be used anywhere else.